src/main/java/core/basesyntax/dao/DataFromCsvFile.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax.dao;
+
+import core.basesyntax.service.FruitTransaction;
+import java.util.List;
+
+public interface DataFromCsvFile {
+    int OPERATION = 0;
+    int FRUIT_NAME = 1;
+    int QUANTITY = 2;
```

Comment:
I dont think we should make this fields as interface constants. This interface should be able to work with different files with different structures